libs/bloom-lfp/src/WinFormsStyleSelect.tsx

```diff
@@ -0,0 +1,121 @@
+import { css } from "@emotion/react";
+import {
+  FormControl,
+  MenuProps,
+  Select,
+  SelectChangeEvent,
+} from "@mui/material";
+import { ThemeProvider, Theme } from "@mui/material/styles";
+import { FunctionComponent, ReactNode } from "react";
+
+import { lightTheme } from "./BloomMuiTheme";
+
+declare module "@mui/material/styles" {
+  // eslint-disable-next-line @typescript-eslint/no-empty-interface
+  interface DefaultTheme extends Theme {}
+}
+
+// This seemed to be the only way to affect the css of the popped up list, since it's a completely
+// separate html element from this component.
+// TODO: migrate away from MUIv4's useStyle usage:
+/*const useStyles = makeStyles(() => ({
+  menuPaper: {
+    maxHeight: 225,
+    padding: 3,
+  },
+}));*/
```

Comment:
Can you delete this code now since its not doing anything?

Reply:
There's style info in there I may still need to add elsewhere.